subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -21,6 +21,10 @@
 #include "ll_feat.h"
 #include "ll_settings.h"
 
+#if defined(CONFIG_BT_CTLR_TX_PWR_DYNAMIC_CONTROL)
```

Comment:
Dont need the conditional compile unless the includes are not implicitly excluded by compiler when not being referenced in following code in this file.